contracts/Bitmask.sol

```diff
@@ -0,0 +1,21 @@
+pragma solidity ^0.4.22;
+
+library Bitmask {
```

Comment:
Probably worth adding basic NatSpec documentation, at the contract level and also for each function.

Reply:
Added at the contract level.  Honestly I think the functions are self explanatory for anyone who should be using this lib

Reply:
Fair